src/components/sections/individual-adopters/Comments.tsx

```diff
@@ -0,0 +1,170 @@
+  width: ${props => (props.theme.screens.md ? "3rem" : "")};
+`;
+
+const Why: React.FC<Icontent> = ({ adopter }) => {
+  return (
+    <ContentSection>
+      <Row>
+        <Img src={`/assets/bullets/${adopter.key}/why.svg`} />
+        <TextBox>
+          <SubHeading>{adopter?.why.heading}</SubHeading>
+          <br />
+          <SubTextGray>{adopter?.why.subtext}</SubTextGray>
+          <SubTextGray>
+            {adopter?.why.list ? (
+              <List>
+                {adopter?.why.list.map((adopter: any) => {
+                  return <li>{adopter}</li>;
+                })}
+              </List>
+            ) : (
+              <></>
+            )}
+          </SubTextGray>
+        </TextBox>
+      </Row>
+    </ContentSection>
+  );
+};
+const How: React.FC<Icontent> = ({ adopter }) => {
+  return (
+    <ContentSection>
+      <Row>
+        <Img src={`/assets/bullets/${adopter.key}/how.svg`} />
+        <TextBox>
+          <SubHeading>{adopter?.how.heading}</SubHeading>
+          <br />
+          <SubTextGray>{adopter?.how.subtext}</SubTextGray>
+          <SubTextGray>
+            {adopter?.how.list ? (
+              <List>
+                {adopter?.how.list.map((adopter: any) => {
+                  return <li>{adopter}</li>;
+                })}
+              </List>
+            ) : (
+              <></>
+            )}
+          </SubTextGray>
+        </TextBox>
+      </Row>
+    </ContentSection>
+  );
+};
+const Benefits: React.FC<Icontent> = ({ adopter }) => {
+  return (
+    <ContentSection>
+      <Row>
+        <Img src={`/assets/bullets/${adopter.key}/benefits.svg`} />
+        <TextBox>
+          <SubHeading>{adopter?.benefits.heading}</SubHeading>
+          <br />
+          <SubTextGray>{adopter?.benefits.subtext}</SubTextGray>
+          <SubTextGray>
+            {adopter?.benefits.list ? (
+              <List>
+                {adopter?.benefits.list.map((adopter: any) => {
+                  return <li>{adopter}</li>;
+                })}
+              </List>
+            ) : (
+              <></>
+            )}
+          </SubTextGray>
+        </TextBox>
+      </Row>
+    </ContentSection>
+  );
+};
+const Other: React.FC<Icontent> = ({ adopter }) => {
+  return adopter.other.heading ? (
+    <ContentSection>
+      <Row>
+        <Img src={`/assets/bullets/${adopter.key}/others.svg`} />
+        <TextBox>
+          <SubHeading>{adopter?.other.heading}</SubHeading>
+          <SubTextGray>
+            {adopter?.other.list ? (
+              <List>
+                {adopter?.other.list.map((adopter: any) => {
+                  return <li>{adopter}</li>;
+                })}
+              </List>
+            ) : (
+              <>
+                <br />
+                Videos:{" "}
+                <a rel="noopener noreferrer" href={adopter?.other.videoUrl}>
+                  {adopter?.other.videoUrl}
+                </a>
+                {adopter?.other.blogUrl ? (
```

Comment:
same

Reply:
The linter was throwing an error that object might be empty or null